scripts/run-and-test.sh

```diff
@@ -1,6 +1,7 @@
-docker stop zed-testserver && docker rm zed-testserver
-docker run -d -p 50051:50051 --name=zed-testserver quay.io/authzed/spicedb:latest serve-testing
-sleep 1
+#! /bin/sh
+
+set -e
```

Comment:
Unfortunately this leaves the spicedb container running if there are test failures. May just need to explicitly capture and return the exit code